core/jobs/batch_jobs/exp_migration_jobs.py

```diff
@@ -180,6 +185,48 @@ def _update_exp_summary(
 
         return updated_exp_summary_model
 
+    @staticmethod
+    def _extract_required_translation_model(
+        exp_translation_models: List[List[
+            translation_models.EntityTranslationsModel]],
+        exp_version: int
+    ) -> List[translation_models.EntityTranslationsModel]:
+        """Extracts all the latets translation models from the given list of
+        translation models.
+
+        Args:
+            exp_translation_models: list(EntityTranslationsModel). The list of
+                given entity translation models.
+            exp_version: int. The latest version number of the exploration.
+        """
+        if not exp_translation_models:
+            return []
+
+        all_models = []
+        for translation_model in list(exp_translation_models[0]):
+            if translation_model.entity_version == exp_version:
+                all_models.append(translation_model)
+        return all_models
+
+    @staticmethod
+    def _generate_new_translation_model(
+            exp_translation_model: translation_models.EntityTranslationsModel
+        ) -> result.Result[translation_models.EntityTranslationsModel]:
```

Comment:
```suggestion
    def _generate_new_translation_model(
        exp_translation_model: translation_models.EntityTranslationsModel
    ) -> result.Result[translation_models.EntityTranslationsModel]:
```

Also, I don't think that `Result` is needed if we only have `Ok` outputs.

Reply:
Done!